src/k8s/pkg/utils/sys.go

```diff
@@ -7,8 +7,8 @@ import (
 	"os/exec"
 )
 
-// runCommand executes a command with a given context.
-// runCommand returns nil if the command completes successfully and the exit code is 0.
+// RunCommand executes a command with a given context.
+// RunCommand returns nil if the command completes successfully and the exit code is 0.
```

Comment:
good catch, but not related with this PR

Reply:
It's best to avoid unrelated changes together, to avoid merge conflicts in the future.